hazelcast/src/main/java/com/hazelcast/durableexecutor/impl/operations/PutResultOperation.java

```diff
@@ -58,6 +65,17 @@ public WaitNotifyKey getNotifiedKey() {
     }
 
     @Override
+    public boolean shouldBackup() {
+        Version version = getNodeEngine().getClusterService().getClusterVersion();
+        return version.isGreaterOrEqual(V3_9);
```

Comment:
If #10010 is merged, `3.8` should be also able to handle a backup (see comment below). wdyt?